### ISSUE TITLE
getting error after updating nanoc & guard nanoc

### ISSUE THREAD

**Issue by BobWalsh** [guard/guard-nanoc#35]
After updating a project's gemfile and bundle updating nanoc (from 3.8.0 --> 4.6.1.) and guard-nanoc (from 1.0.3 --> 2.1.1) I get the following. Using ruby 2.3.1p112 and Rails 4.2.7.1. 

Returning to the old versions of each gem fixes the problem. Any idea what's the problem/how to resolve? 

---
```
Bundler::GemRequireError: There was an error while trying to load the gem 'guard-nanoc'.
Gem Load Error is: uninitialized constant Nanoc::CLI
Backtrace for gem load error is:
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/guard-nanoc-2.1.1/lib/guard-nanoc.rb:3:in `<top (required)>'
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/bundler-1.14.3/lib/bundler/runtime.rb:91:in `require'
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/bundler-1.14.3/lib/bundler/runtime.rb:91:in `block (2 levels) in require'
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/bundler-1.14.3/lib/bundler/runtime.rb:86:in `each'
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/bundler-1.14.3/lib/bundler/runtime.rb:86:in `block in require'
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/bundler-1.14.3/lib/bundler/runtime.rb:75:in `each'
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/bundler-1.14.3/lib/bundler/runtime.rb:75:in `require'
/Users/bobwalsh/.rvm/gems/ruby-2.3.1@orangeqc/gems/bundler-1.14.3/lib/bundler.rb:107:in `require'
/Users/bobwalsh/orangeqc/config/application.rb:7:in `<top (required)>'
/Users/bobwalsh/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/Users/bobwalsh/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/Users/bobwalsh/orangeqc/config/environment.rb:2:in `<top (required)>'
/Users/bobwalsh/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/Users/bobwalsh/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/Users/bobwalsh/orangeqc/config.ru:11:in `block in <main>'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/lib/nack/builder.rb:4:in `instance_eval'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/lib/nack/builder.rb:4:in `initialize'
/Users/bobwalsh/orangeqc/config.ru:1:in `new'
/Users/bobwalsh/orangeqc/config.ru:1:in `<main>'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/lib/nack/server.rb:51:in `eval'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/lib/nack/server.rb:51:in `load_config'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/lib/nack/server.rb:42:in `initialize'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/lib/nack/server.rb:12:in `new'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/lib/nack/server.rb:12:in `run'
/Users/bobwalsh/Library/Application Support/Pow/Versions/0.5.0/node_modules/nack/bin/nack_worker:4:in `<main>'
```

**Comment by ddfreyne** [comment]
Very strange… I cannot reproduce the problem, but I can see that the code could cause this issue (it is missing a `require 'nanoc/cli'`). Fix coming up.

**Comment by ddfreyne** [comment]
@BobWalsh Could you verify that current master (which has the fix 251d0013c836b776a369464d5a9a9fa4e8480b9d) works for you? To do so, change the `guard-nanoc` line in the Gemfile to say

```
gem 'guard-nanoc', github: 'guard/guard-nanoc'
```

then run `bundle update guard-nanoc`, and re-try.
